### PR TITLE
Update INSTALL.md

### PR DESCRIPTION
**Pull request by redsnout**
Added a quick note for installing pySerial on macOS

### CODE REVIEW

**Comment by ladislas** [comment]
thanks @redsnout for the PR!

looking at the context of the changes, it appears that my python instructions are a bit out of date.

Python 2 has been deprecated, Python 3 should be used instead. Could you fix line 123 by removing the `python@2`?

Also installing pyserial should be done this way now:

```bash
python3 -m pip install -U --user pyserial
```

Could you update line 129 with this?

**Comment by redsnout** [comment]
@ladislas Thanks for the feedback. I have updated with the two additional things you have mentioned.

File: INSTALL.md [hunk 1]
```diff
@@ -120,13 +120,17 @@ To upload the program, we need to reset the Arduino board. This is done using a
...
 ```
 
 Then install `pySerial`:
 
 ```Bash
-$ pip install pyserial & pip2 install pyserial
+$ python3 -m pip install -U --user pyserial
+```
+To install pySerial on OS X: Download pyserial from [PyPI](https://pypi.org/project/pyserial/#files), unpackage the download, and CD into the folder. Then run the following command:
+```Bash
```
Comment (ladislas) [inline INSTALL.md:132]: this part must be removed, it's not needed.

**Comment by ladislas** [comment]
thanks @redsnout -- I've added a comment, can you remove the "To install pySerial on OS X...", it is not needed anymore

**Comment by redsnout** [comment]
Sorry for the confusion, we should be good now.

**Review by ladislas — APPROVED** [review]

**Comment by ladislas** [comment]
Thanks a lot! 👍

**Pull request merged**